### PR TITLE
Included yt_dlp python package and ffmpeg binary to download audio from YouTube

### PR DESCRIPTION
**Pull request by roshansthomas**
*Issue #33 

*Description of changes:*
1. Removed dependency on pytube and included yt_dlp python package and ffmpeg binary to download audio from YouTube.
2. Introduced the use of aws cloudformation package cli to create artefacts as part of publish.sh

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: layers/qbusiness_boto3_model/model/qbusiness/2023-11-27/service-2.json
Comment (rstrahan) [inline layers/qbusiness_boto3_model/model/qbusiness/2023-11-27/service-2.json:1]: This should not be needed..  this release doesn't use qbusiness

File: cfn-templates/msindexer.yaml [hunk 1]
```diff
@@ -65,33 +65,44 @@ Resources:
...
-            - '<PYTUBELLAYER_ZIPFILE>'
-      Description: Layer with pytube package required to download YT Videos
-      LayerName: pytube-llayer
+      Content: ../layers/yt_dlp
+      CompatibleRuntimes:
+        - python3.11      
+      Description: Layer with yt_dlp package required to download YT Videos
       LicenseInfo: MIT
+
+#  QBusinessModelLayer:
```
Comment (rstrahan) [inline cfn-templates/msindexer.yaml]: We don't need QBisinessLayer for this release, right?

File: cfn-templates/msindexer.yaml [hunk 2]
```diff
@@ -65,33 +65,44 @@ Resources:
...
-          - !Ref PytubeLambdalayer
-      Code:
-        S3Bucket: '<ARTIFACT_BUCKET_TOKEN>'
-        S3Key: !Join
-          - ''
-          - - '<ARTIFACT_PREFIX_TOKEN>'
-            - '<YTINDEXER_ZIPFILE>'
+          - !Ref YTDLPLambdalayer
+          - !Ref FFMPEGLambdalayer
+#          - !Ref QBusinessModelLayer
```
Comment (rstrahan) [inline cfn-templates/msindexer.yaml]: Not using QBusiness in this release

File: layers/qbusiness_boto3_model/model/qbusiness/2023-11-27/service-2.json
Comment (roshansthomas) [reply layers/qbusiness_boto3_model/model/qbusiness/2023-11-27/service-2.json:1]: Agree. Removed in future commit

File: cfn-templates/msindexer.yaml (hunk 1 above)
Comment (roshansthomas) [reply cfn-templates/msindexer.yaml]: No, I have removed the commented line to avoid confusion

File: cfn-templates/msindexer.yaml (hunk 2 above)
Comment (roshansthomas) [reply cfn-templates/msindexer.yaml]: Agreed. Removing commented line to avoid confusion

**Comment by rstrahan** [comment]
Duplicated in #35

**Pull request closed without merging**